### PR TITLE
[DEV-132563] Add project to public repository

### PR DESCRIPTION
**Pull request by andresguedes**
Adding SDK source code to a public repository

### CODE REVIEW

File: paywithmybank-android-sdk/publish-remote.gradle [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        publications {
+            release(MavenPublication) {
+                // The coordinates of the library, being set from variables that
+                // we'll setup later
+                groupId PUBLISH_GROUP_ID
+                artifactId PUBLISH_ARTIFACT_ID
+                version PUBLISH_VERSION
+
+                // Two artifacts, the `aar` (or `jar`) and the sources
+//                if (project.plugins.findPlugin("com.android.library")) {
```
Comment (marcosrivereto) [inline paywithmybank-android-sdk/publish-remote.gradle:35]: Maybe, remove this comments

**Review by marcosrivereto — APPROVED** [review]

File: paywithmybank-android-sdk/publish-remote.gradle (hunk 1 above)
Comment (andresguedes) [reply paywithmybank-android-sdk/publish-remote.gradle:35]: @marcosrivereto this block of code creates the .jar and the .aar files, but when I try to run in our current project, it doesn't work well. I think we'll need to update the current app project that uses the SDK module to this works fine!

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
-# trustly-android
\ No newline at end of file
+# Trustly Android SDK
+
+This repository contains all source code from our Android SDK, like interfaces and views that creates the integration with Trustly.
```
Comment (lukevance) [inline README.md:3]: "This repository contains the source code for the Trustly Android SDK, including interfaces and views that integrated with the web-based Trustly user interface module."

**Review by pedroabdenor — APPROVED** [review]
👍

**Review by lukevance — APPROVED** [review]
Awesome, looks good!

**Review by lukevance — APPROVED** [review]
👓 👍 LGTM!

**Review by pedroabdenor — APPROVED** [review]
👍

File: README.md
Comment (lukevance) [inline README.md:1]: Perfect!

File: trustly-android-sdk/src/main/java/net/trustly/android/sdk/interfaces/Trustly.java [hunk 3]
```diff
@@ -0,0 +1,138 @@
+/*  ___________________________________________________________________________________________________________
+ *
+ *    TRUSTLY CONFIDENTIAL AND PROPRIETARY INFORMATION
```
Comment (lukevance) [inline trustly-android-sdk/src/main/java/net/trustly/android/sdk/interfaces/Trustly.java:3]: Can you remove this? And any other sections where it is present?

File: trustly-android-sdk/src/main/java/net/trustly/android/sdk/interfaces/TrustlyCallback.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
+/*  ___________________________________________________________________________________________________________
+ *
+ *    TRUSTLY CONFIDENTIAL AND PROPRIETARY INFORMATION
```
Comment (lukevance) [inline trustly-android-sdk/src/main/java/net/trustly/android/sdk/interfaces/TrustlyCallback.java:3]: Here too

File: trustly-android-sdk/src/main/java/net/trustly/android/sdk/util/UrlUtils.java [hunk 5]
```diff
@@ -0,0 +1,91 @@
+/*  ___________________________________________________________________________________________________________
+ *
+ *    TRUSTLY CONFIDENTIAL AND PROPRIETARY INFORMATION
+ *  ___________________________________________________________________________________________________________
+ *
+ *      Copyright (c) 2012 - 2020 Trustly
+ *      All Rights Reserved.
```
Comment (lukevance) [inline trustly-android-sdk/src/main/java/net/trustly/android/sdk/util/UrlUtils.java:7]: Also here

**Review by lukevance — CHANGES_REQUESTED** [review]
Only request, please remove all the "CONFIDENTIAL" and "Copyright" language. Since this is now open source under the MIT license, this language is conflicting.

**Review by lukevance — CHANGES_REQUESTED** [review]
Only request, please remove all the "CONFIDENTIAL" and "Copyright" language. Since this is now open source under the MIT license, this language is conflicting.

**Review by lukevance — APPROVED** [review]
Perfect! 💯

**Comment by rhaissatrustly** [comment]
Approved.

**Pull request merged**